app/src/main/java/net/wetfish/wetfish/adapters/ExifDataAdapter.java

```diff
@@ -291,6 +299,47 @@ public void bind(final FileExifData exifData, final int position) {
                 mExifDataTag.setText(exifData.getExifDataTag());
                 mExifDataValue.setText(exifData.getExifDataValue());
 
+                // Check to see if an editedExifDataList has been instantiated
+                if (!mEditedExifDataListInstantiated) {
```

Comment:
instead of using this boolean i would recommend using ArrayList.isEmpty(), or if you do not instantiate the object above you could use mEditedExifDataList == null, then instantiate immediately after.

Reply:
Good catch, the boolean is no longer necessary after the code refactoring.

The change will be in the next commit.